src/instrument/callbacks/spec_data_file_writer.py

```diff
@@ -73,17 +73,18 @@ def newSpecFile(title, scan_id=None, RE=None):
 """The SPEC file writer object."""
 
 # make the SPEC file in current working directory (assumes is writable)
-_path = pathlib.Path().cwd()
-specwriter.newfile(_path / specwriter.spec_filename)
+specwriter.newfile(specwriter.spec_filename)
 
 if "SPEC_DATA_FILES" in iconfig:
     RE.subscribe(specwriter.receiver)  # write data to SPEC files
+    logger.info("SPEC data file: %s", specwriter.spec_filename.resolve())
```

Comment:
Not seeing this log message on startup.

```py
...
is not notebook



W Wed-15:57:17 - APS DM setup file does not exist: '/home/dm/etc/dm.setup.sh'
I Wed-15:57:17 - /home/prjemian/Documents/projects/prjemian/model_instrument/src/instrument/utils/ophyd_setup.py
I Wed-15:57:17 - using ophyd control layer: 'pyepics'

In [2]: 
```

Expect the SPEC data file name to appear in the logs right after the ophyd control layer.